Source/Views/StackViewController.swift

```diff
@@ -69,4 +73,58 @@ final class StackViewController: UIViewController {
         stackView.addArrangedSubview(childViewController.view)
         childViewController.didMove(toParentViewController: self)
     }
+    
+    /// A method to access the scrollView from outside and scroll to a given target ViewController. 
+    /// If no relativePosition parameter is provided, it will try to center the target.
+    ///
+    /// - Parameters:
+    ///   - target: Each child should have a meta representing `TabConfig`
+    ///   - relativePosition: An optional parameter to specify the alignement of the target. Depending of 
+    ///     the stackViews' orientation the relativePosition values have different meaning, e.g. top or 
+    ///     left.
+    public func scrollToViewController(target targetViewController: UIViewController,
```

Comment:
as you mentioned this won't work for nested stacks you need to use  `func convert(CGRect, to: UIView?)` to convert a view frame to the equivalent in this VC's view.
However this will be enough for example for vertical stacks with vertical stacks but might break for example with a vertical stack that contains an horizontal and needs to be scrolled to:
- the right point vertically in the main stack
- the right point horizontally in the inner stack

I have a solution in mind but I need to write down some pseudo code,  maybe you guys can go ahead and eventually ping me if need help (I'm a bit busy and I might forget about it, let's see)